pypeit/reduce.py

```diff
@@ -1649,9 +1685,16 @@ def joint_skysub(self, skymask=None, update_crmask=True, trim_edg=(0,0),
                                                              pos_mask=(not self.ir_redux), show_fit=show_fit)
             # Update the ivar image used in the sky fit
             msgs.info("Updating sky noise model")
-            var = np.abs(self.global_sky - np.sqrt(2.0) * np.sqrt(self.sciImg.rn2img)) + self.sciImg.rn2img
-            var = var + adderr ** 2 * (np.abs(self.global_sky)) ** 2
-            model_ivar = utils.inverse(var)
+            # Choose the highest counts out of sky and object
+            counts = self.global_sky + np.clip(self.sciImg.image-self.global_sky, 0, None)
```

Comment:
I've given this some thought, and I think this should be
```counts = self.global_sky```
The reason is because this variance model is only needed for the skysub (the object regions don't matter in this case), so adding a contribution from possible object photons (the `np.clip()` part) will only act to bias the noise model high slightly.

Reply:
Thanks, @rcooke-ast !!